config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-v1beta1-release-1.8.yaml

```diff
@@ -11,7 +11,7 @@ periodics:
   extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.8
+      base_ref: release-1.9
```

Comment:
this change seems like a mistake? it's in the file testing release-1.8

Reply:
good catch

this is why my original attempt to do this kept a mutable "-minus-1" naming for these, handling all these changes with renaming the files and names and tabs every CAPZ release is going to be hard

Reply:
would be great if we could generate these... just like we generate the k/k azure jobs

Reply:
I fixed both mistakes by reverting to the original files, renaming the -1.8 job to -1.9, renaming the -1.7 job to -1.8, and doing the appropriate replacements within each.

Reply:
You make it sound easy!